src/Microsoft.OpenApi.Readers/ParseNodes/JsonPointerExtensions.cs

```diff
@@ -47,9 +47,9 @@ public static YamlNode Find(this JsonPointer currentpointer, YamlNode baseYamlNo
 
                 return pointer;
             }
-            catch (Exception ex)
+            catch (Exception)
             {
-                throw new ArgumentException("Failed to dereference pointer", ex);
+                return null;
```

Comment:
I'm still a little nervous about this change, but let's try it and see if it causes any issues.